department-of-veterans-affairs/vets-website
# Pension 70328 mvp applicant info ssn and va file number update

## Summary
On the Pension Benefits application(21P-527EZ), update the 'Applicant Information' page

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Select 'Start your application without signing in'
3. Verify the page 'Applicant Information' is in the correct location, per the designs, and includes a SSN field, a file claims yes/no radio question, and a VA file number field
4. Verify that the SSN field is validated for format
5. Verify the the VA file number field is validated for format
6. Verify the SSN is required

## Related issue(s)
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/70328)
[Epic in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349)

## Testing done

## Screenshots

### Desktop
| Before | After | After (Valid) |
| ------ | ------ | ------ | 
|![localhost_3001_pension_application_527EZ_applicant_information](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/ecd88288-45ea-457f-b1ce-a53283a1cc4b)|![localhost_3001_pension_application_527EZ_](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/7612df15-32d9-4a73-9aab-8a327f863bf3)|![localhost_3001_pension_application_527EZ_ (1)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/afd521b5-5b8e-42ed-8b4f-228890208837)



## What areas of the site does it impact?
Pension Benefits Application

## Acceptance criteria
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/70328)

### Quality Assurance & Testing

### Error Handling


### Authentication
